java-source/src/main/java/com/example/flow/ExampleFlow.java

```diff
@@ -84,7 +84,7 @@ public ProgressTracker getProgressTracker() {
         @Override
         public SignedTransaction call() throws FlowException {
             // Obtain a reference to the notary we want to use.
-            final Party notary = getServiceHub().getNetworkMapCache().getNotaryNodes().get(0).getNotaryIdentity();
+            final Party notary = getServiceHub().getNetworkMapCache().getAnyNotary();
```

Comment:
Change it to `getNotary()`, Andrius is working now on removing `getAnyNotary` function.

Reply:
But `getNotary()` is defined as:
`fun getNotary(name: CordaX500Name): Party?`
which `name` are we meant to pass in?

Reply:
Ah, sorry I forgot about that, you could choose the first one from notaries list. However it should be commented similarly to `FlowCookbook` on Andrius PR.